src/main.cpp

```diff
@@ -40,7 +40,7 @@ main(int argc, char **argv)
       homestr = ".";
     app.processEvents();
     std::string localDir = std::string(home);
-    app.indexDirectory(localDir.c_str());
+    app.indexDirectory(localDir.c_str()); 
```

Comment:
Not a big deal for this PR, but you generally want to give your diff a once over review to avoid changing things like this (you added a space to the end of a line).